File: Blanquita-Backend/app/Models/Usuario/UsuarioLogIn.py
from pydantic import BaseModel


class UsuarioLogin(BaseModel):
    Ci: str
    Clave: str


class UsuarioLoginResponse(BaseModel):
    access_token: str
    IdUsuario: int
    IdRol: int
    NombreRol: str
    PrimerNombre: str
    ApellidoPaterno: str

File: Blanquita-Frontend/src/models/Usuario/UsuarioLogIn.js
export const UsuarioLogin = (Ci, Clave) => ({
  Ci,
  Clave
});

export const UsuarioLoginResponse = (data) => ({
  access_token: data.access_token,
  IdUsuario: data.IdUsuario,
  IdRol: data.IdRol,
  NombreRol: data.NombreRol,
  PrimerNombre: data.PrimerNombre,
  ApellidoPaterno: data.ApellidoPaterno
});

export const UsuarioSesion = (data) => ({
  IdUsuario: data.IdUsuario,
  IdRol: data.IdRol,
  NombreRol: data.NombreRol,
  PrimerNombre: data.PrimerNombre,
  ApellidoPaterno: data.ApellidoPaterno
});